tests/config.c

```diff
@@ -370,13 +371,30 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
             // Trace both allocations & frees from shared memory segment.
             platform_enable_tracing_shm_ops();
          }
+         // Parameter should only be used with --use-shmem argument.
+         config_has_option("fork-child")
+         {
+            for (uint8 cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+               cfg[cfg_idx].fork_child = TRUE;
+            }
+         }
+
+         // Some tests that fork multiple child processes may need
+         // debugging. Use this arg to wait-for-gdb looping behaviour.
+         config_has_option("wait-for-gdb")
+         {
+            for (uint8 cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+               cfg[cfg_idx].wait_for_gdb = TRUE;
+            }
+         }
 
          config_set_uint64("key-size", cfg, max_key_size) {}
          config_set_uint64("data-size", cfg, message_size) {}
 
          // Test-execution configuration parameters
          config_set_uint64("seed", cfg, seed) {}
          config_set_uint64("num-inserts", cfg, num_inserts) {}
+         config_set_uint64("num-forked-processes", cfg, num_forked_processes) {}
```

Comment:
I think "num-processes" would be better.

Reply:
Changed as suggested.

Also, discovered that there was no invocation of a test with this argument. ... Good!

Adjusted `test.sh/run_slower_forked_process_tests()`, to add in a call to `unit/splinterdb_forked_child_test` with this new `--num-processes` argument, for improved coverage (of this new config option).